doc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,126 @@
+# Release Template
+
+> short tl;dr; of the release
+
+# 🗺 What's left for release
+
+<List of items with PRs and/or Issues to be considered for this release>
+
+# 🚢 Estimated shipping date
+
+<Date this release will ship on if everything goes to plan>
```

Comment:
I added this so that people can get a vague idea of when this release might be available... 👍 👎 ?

Reply:
I think specifying the estimated week would be sufficient and helpful, since this is designed to go in 6 week cycles. I don't think any more granular than that is necessary unless the release cycle ends up targeting specific days of the week.